Compile errors with Tawaki

#3199 brings some new errors: 
 - ```conf/conf_tests_coverity.xml```
 - ```BUILD AIRCRAFT=Ark_Quad, TARGET ap```

```
arch/chibios/modules/actuators/actuators_pwm_arch.c:354:13: error: 'PWMD4' undeclared (first use in this function); did you mean 'PWMD5'?
  354 |   pwmStart(&PWMD4, &pwmcfg4);
      |             ^~~~~
      |             PWMD5
```

--- a/sw/airborne/boards/lisa_mx/chibios/v2.1/board.h
+++ b/sw/airborne/boards/lisa_mx/chibios/v2.1/board.h
@@ -1173,7 +1173,11 @@
 #endif
 
 #ifndef USE_PWM_TIM4
+#ifdef USE_SERVOS_7AND8
 #define USE_PWM_TIM4 1
+#else
+#define USE_PWM_TIM4 0
+#endif
 #endif
 
 #ifndef USE_PWM_TIM5

--- a/sw/airborne/boards/lisa_mxs/chibios/v1.0/board.h
+++ b/sw/airborne/boards/lisa_mxs/chibios/v1.0/board.h
@@ -1172,7 +1172,11 @@
 #endif
 
 #ifndef USE_PWM_TIM4
+#ifdef USE_SERVOS_7AND8
 #define USE_PWM_TIM4 1
+#else
+#define USE_PWM_TIM4 0
+#endif
 #endif
 
 #ifndef USE_PWM_TIM5

